From f6c404ecd3ef83f947f31e18bd9b85078f415d6e Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Tue, 18 Mar 2014 09:18:17 -0700
Subject: [PATCH 195/456] media: xilinx: vip: Remove _TIMING_ from register
 definition

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Remove _TIMING_ from register definitions. Change the names in xilinx-tpg.c
accordingly.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 86d39534db3e10993aefc13ea7d57342bd98900b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-tpg.c |    6 ++--
 drivers/media/platform/xilinx/xilinx-vip.h |   36 ++++++++++++++--------------
 2 files changed, 21 insertions(+), 21 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-tpg.c b/drivers/media/platform/xilinx/xilinx-tpg.c
index 7853cb4..4c2822f 100644
--- a/drivers/media/platform/xilinx/xilinx-tpg.c
+++ b/drivers/media/platform/xilinx/xilinx-tpg.c
@@ -81,9 +81,9 @@ static int xtpg_s_stream(struct v4l2_subdev *subdev, int enable)
 		return 0;
 	}
 
-	xvip_write(&xtpg->xvip, XVIP_TIMING_ACTIVE_SIZE,
-		   (height << XVIP_TIMING_ACTIVE_VSIZE_SHIFT) |
-		   (width << XVIP_TIMING_ACTIVE_HSIZE_SHIFT));
+	xvip_write(&xtpg->xvip, XVIP_ACTIVE_SIZE,
+		   (height << XVIP_ACTIVE_VSIZE_SHIFT) |
+		   (width << XVIP_ACTIVE_HSIZE_SHIFT));
 
 	xvip_write(&xtpg->xvip, XTPG_PATTERN_CONTROL, 0x00001029);
 	xvip_write(&xtpg->xvip, XTPG_MOTION_SPEED, 1);
diff --git a/drivers/media/platform/xilinx/xilinx-vip.h b/drivers/media/platform/xilinx/xilinx-vip.h
index 48842ad..13ab1b9 100644
--- a/drivers/media/platform/xilinx/xilinx-vip.h
+++ b/drivers/media/platform/xilinx/xilinx-vip.h
@@ -48,24 +48,24 @@
 #define XVIP_CTRL_VERSION_INTERNAL_SHIFT	0
 
 /* Xilinx Video IP Timing Registers */
-#define XVIP_TIMING_ACTIVE_SIZE			0x0020
-#define XVIP_TIMING_ACTIVE_VSIZE_MASK		(0x7ff << 16)
-#define XVIP_TIMING_ACTIVE_VSIZE_SHIFT		16
-#define XVIP_TIMING_ACTIVE_HSIZE_MASK		(0x7ff << 0)
-#define XVIP_TIMING_ACTIVE_HSIZE_SHIFT		0
-#define XVIP_TIMING_OUTPUT_ENCODING		0x0028
-#define XVIP_TIMING_OUTPUT_NBITS_8		(0 << 4)
-#define XVIP_TIMING_OUTPUT_NBITS_10		(1 << 4)
-#define XVIP_TIMING_OUTPUT_NBITS_12		(2 << 4)
-#define XVIP_TIMING_OUTPUT_NBITS_16		(3 << 4)
-#define XVIP_TIMING_OUTPUT_NBITS_MASK		(3 << 4)
-#define XVIP_TIMING_OUTPUT_NBITS_SHIFT		4
-#define XVIP_TIMING_VIDEO_FORMAT_YUV422		(0 << 0)
-#define XVIP_TIMING_VIDEO_FORMAT_YUV444		(1 << 0)
-#define XVIP_TIMING_VIDEO_FORMAT_RGB		(2 << 0)
-#define XVIP_TIMING_VIDEO_FORMAT_YUV420		(3 << 0)
-#define XVIP_TIMING_VIDEO_FORMAT_MASK		(3 << 0)
-#define XVIP_TIMING_VIDEO_FORMAT_SHIFT		0
+#define XVIP_ACTIVE_SIZE			0x0020
+#define XVIP_ACTIVE_VSIZE_MASK			(0x7ff << 16)
+#define XVIP_ACTIVE_VSIZE_SHIFT			16
+#define XVIP_ACTIVE_HSIZE_MASK			(0x7ff << 0)
+#define XVIP_ACTIVE_HSIZE_SHIFT			0
+#define XVIP_ENCODING				0x0028
+#define XVIP_ENCODING_NBITS_8			(0 << 4)
+#define XVIP_ENCODING_NBITS_10			(1 << 4)
+#define XVIP_ENCODING_NBITS_12			(2 << 4)
+#define XVIP_ENCODING_NBITS_16			(3 << 4)
+#define XVIP_ENCODING_NBITS_MASK		(3 << 4)
+#define XVIP_ENCODING_NBITS_SHIFT		4
+#define XVIP_ENCODING_VIDEO_FORMAT_YUV422	(0 << 0)
+#define XVIP_ENCODING_VIDEO_FORMAT_YUV444	(1 << 0)
+#define XVIP_ENCODING_VIDEO_FORMAT_RGB		(2 << 0)
+#define XVIP_ENCODING_VIDEO_FORMAT_YUV420	(3 << 0)
+#define XVIP_ENCODING_VIDEO_FORMAT_MASK		(3 << 0)
+#define XVIP_ENCODING_VIDEO_FORMAT_SHIFT	0
 
 /**
  * struct xvip_device - Xilinx Video IP device structure
-- 
1.7.5.4

